frontend/src/components/graph.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { VictoryStack, VictoryAxis, VictoryArea, VictoryTheme, VictoryChart, VictoryLine, LineSegment, VictoryLabel} from 'victory';
+import {
+    getResponseByTractIDAndYear
+} from "../utils/apiWrapper";
+
+class Graph extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            data: [],
+            x_labels: [],
+            y_labels: [],
+            tract_id: this.props.tract_id,
+            year: 2010
+        };
+    }
+
+    async componentDidMount() {
+        const response = await getResponseByTractIDAndYear(this.state.tract_id, this.state.year);
+        if (!response.data.result.response_rates["0"]) {
+          return;
+        }
+        let rates_dict = {};
+        rates_dict = response.data.result.response_rates["0"].rates[this.state.tract_id]
+        const rates_list = [];
+        for (var key in rates_dict) {
+            rates_list.push({"x": rates_dict[key][1], "y": rates_dict[key][0]});
+        }
+        console.log(rates_list);
+        const STEPS = 5
```

Comment:
should this be declared on top of this page since it is a magic number?

Reply:
resolved